docs/rst/tutorials/cloud/secure_router/secure_router.rst

```diff
@@ -160,42 +192,34 @@ The tutorial workspace will have the following structure at the end of the proje
 
```

Comment:
Is the previous step (create the workspace) needed always or only if you are using docker and you want to export the workspace?

Reply:
Actually the workspace is created in the ROS2 tutorial so we are using that one and adding our files for the router